quantum/plugins/qrack/CMakeLists.txt

```diff
@@ -0,0 +1,65 @@
+set(LIBRARY_NAME xacc-qrack)
+
+file(GLOB SRC
+          *.cpp
+          accelerator/*.cpp)
+
+usfunctiongetresourcesource(TARGET ${LIBRARY_NAME} OUT SRC)
+usfunctiongeneratebundleinit(TARGET ${LIBRARY_NAME} OUT SRC)
+
+add_library(${LIBRARY_NAME} SHARED ${SRC})
+
+target_include_directories(${LIBRARY_NAME}
+                             PUBLIC .
+                                    ./accelerator
+                                    ./accelerator/src/include
+                                    ${CMAKE_SOURCE_DIR}/tpls/eigen)
```

Comment:
Are you using `eigen` in qrack?

Reply:
No, I am not. This was left behind in migrating from the Quantum++ plugin, and I will remove it.

Reply:
Done.